### ISSUE TITLE
Don’t work! 

### ISSUE THREAD

**Issue by taraldsen** [easyhomesrc/homey_easylighting#11]
It dont work in Homey beta. Any solution on that?

**Comment by maginawin** [comment]
Hi @taraldsen ,

Do you mean that the App cannot works with Homey Bridge?

**Comment by taraldsen** [comment]
Not working with Homey pro beta.
[Image.jpeg]
________________________________
Fra: Maginawin ***@***.***>
Sendt: Wednesday, December 8, 2021 2:36:09 AM
Til: easyhomesrc/homey_easylighting ***@***.***>
Kopi: Bjørnar Taraldsen ***@***.***>; Assign ***@***.***>
Emne: Re: [easyhomesrc/homey_easylighting] Don’t work! (Issue #11)


Assigned #11<https://github.com/easyhomesrc/homey_easylighting/issues/11> to @taraldsen<https://github.com/taraldsen>.

**Comment by taraldsen** [comment]
More explanation. The zigbee switch 4 ch wall did work son time, but not anymore. ![image](https://user-images.githubusercontent.com/4910888/145162413-3d705dd9-b03e-4b53-a3c7-ed67d9763ecc.jpeg)

**Comment by maginawin** [comment]
Hi @taraldsen ,

I tested the same device and it works.

How far is between your switch  and the Homey Pro beta? Is there a wall between them?

When the switch is not working, will the green LED light on after you press the button on the switch? 

Will it still not work after you bring the switch close to the Homey Pro beta?

**Comment by taraldsen** [comment]
How far is between your switch and the Homey Pro beta?   Its maby 3-4 meter and no wall between.

When the switch is not working, will the green LED light on after you press the button on the switch? Yes it blinks one time.

Will it still not work after you bring the switch close to the Homey Pro beta? No

I was uninstall the app now and started over.. but nothing happend..    it was working 2 weeks ago.  When it «works» its on and off in same button. And takes loong time before somthing happend.    The flows for switch working… but som fault on devise… if i do all the right things.  😊

Med vennlig hilsen

Bjørnar Taraldsen

Telefon 948 97 829
E-post: ***@***.******@***.***>

***@***.***

M. Carlsen & Sønn AS

Adresse: Storlinna 88, 2760 Brandbu
Kontakt: 613 36 880, ***@***.******@***.***>
Org. Nr. 998540860
Faktura: ***@***.******@***.***>
Web: www.mcarlsen.no<http://www.mcarlsen.no/>


Fra: Maginawin ***@***.***>
Sendt: onsdag 8. desember 2021 09:40
Til: easyhomesrc/homey_easylighting ***@***.***>
Kopi: Bjørnar Taraldsen ***@***.***>; Mention ***@***.***>
Emne: Re: [easyhomesrc/homey_easylighting] Don’t work! (Issue #11)


Hi @taraldsen<https://github.com/taraldsen> ,

I tested the same device and it works.

How far is between your switch and the Homey Pro beta? Is there a wall between them?

When the switch is not working, will the green LED light on after you press the button on the switch?

Will it still not work after you bring the switch close to the Homey Pro beta?

**Comment by maginawin** [comment]
Hi @taraldsen ,

It looks like the device is working.

What flows have you created for it? 

You mean that some flows sometimes work, sometimes the flows don't work, right?

**Comment by taraldsen** [comment]
Yes.. if I trigger flows in homey it works.. when I hit a button , it turns on and off the lights..  I made 8 flows with different tasks.. import fom hue. Maby that’s the problem… [Image.jpeg]
________________________________
Fra: Maginawin ***@***.***>
Sendt: Wednesday, December 8, 2021 10:39:40 AM
Til: easyhomesrc/homey_easylighting ***@***.***>
Kopi: Bjørnar Taraldsen ***@***.***>; Mention ***@***.***>
Emne: Re: [easyhomesrc/homey_easylighting] Don’t work! (Issue #11)


Hi @taraldsen<https://github.com/taraldsen> ,

It looks like the device is working.

What flows have you created for it?

You mean that some flows sometimes work, sometimes the flows don't work, right?

**Comment by maginawin** [comment]
Hi @taraldsen ,

Sorry, I can't see the image, please send to my email sunricher_src@163.com or upload it at https://github.com/easyhomesrc/homey_easylighting/issues/11
